cmd/monaco/runner/runner.go

```diff
@@ -76,7 +80,9 @@ Examples:
     monaco deploy service.yaml -e dev`,
 
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			fileBasedLogging := featureflags.LogToFile.Enabled() || support.SupportArchive
+			cmd.Context().Value(support.SupportArchive{}).(*support.SupportArchive).Value = supportArchive
```

Comment:
let's also make this a detail in the `support` package

Reply:
done